### PR TITLE
add filter to content label in admin ui

### PR DESCRIPTION
**Pull request by romancandlethoughts**
added filter to the word "content" that appears in the field label and description.

### CODE REVIEW

File: templates/field.php [hunk 1]
```diff
@@ -10,7 +10,12 @@
 	<input type="hidden" class="fm-element zone-name" name="<?php echo esc_attr( $this->get_form_name() ); ?>" value="" />
 
 	<div class="zone-search-wrapper">
-		<label for="<?php echo esc_attr( $this->get_element_id() ); ?>_recent"><?php esc_html_e( 'Add Recent Content', 'fm-zones' ); ?></label><br>
+		<label for="<?php echo esc_attr( $this->get_element_id() ); ?>_recent"><?php
+			printf(
+				'%1$s %2$s',
+				esc_html__( 'Add Recent', 'fm-zones' ),
```
Comment (mboynes) [inline templates/field.php]: This should be "Add Recent %s" and the sprintf should wrap that. There should then be an `esc_html()` around that.

File: templates/field.php [hunk 2]
```diff
@@ -20,9 +25,20 @@
 	</div>
 
 	<div class="zone-search-wrapper">
-		<label for="<?php echo esc_attr( $this->get_element_id() ); ?>_search"><?php esc_html_e( 'Search for content', 'fm-zones' );?></label>
+		<label for="<?php echo esc_attr( $this->get_element_id() ); ?>_search"><?php
+			printf(
```
Comment (mboynes) [inline templates/field.php]: same as above

File: templates/field.php [hunk 3]
```diff
@@ -20,9 +25,20 @@
...
+				'%1$s %2$s',
+				esc_html__( 'Search for', 'fm-zones' ),
+				apply_filters( 'fm-zones-content-label', esc_html__( 'content', 'fm-zones' ) )
+			); ?></label>
 		<input type="text" class="zone-post-search" id="<?php echo esc_attr( $this->get_element_id() ); ?>_search" <?php echo $this->get_element_autocomplete_attributes(); ?> />
-		<p class="description"><?php esc_html_e( 'Enter a term or phrase in the text box above to search for and add content to this zone.', 'fm-zones' ); ?></p>
+		<p class="description"><?php
+			printf(
```
Comment (mboynes) [inline templates/field.php]: same

**Comment by romancandlethoughts** [comment]
@mboynes

**Comment by mboynes** [comment]
:+1:

**Pull request merged**